entry_types/scrolled/package/spec/frontend/usePhonePlatform/inEditorPreview-spec.js

```diff
@@ -0,0 +1,36 @@
+
+import {PhonePlatformProvider} from 'frontend';
+import {usePhonePlatform} from 'frontend/usePhonePlatform';
+import {loadInlineEditingComponents} from 'frontend/inlineEditing';
+
+import {renderHook, act} from '@testing-library/react-hooks';
+
+import '@testing-library/jest-dom/extend-expect'
+import {tick} from "support";
+
+
+describe('usePhonePlatform', () => {
+  beforeAll(loadInlineEditingComponents);
```

Comment:
**praise**: Great that you found out how to cover both the editor and frontend parts in the specs. :+1:

Reply:
Thanks :+1: